module/batch-action/src/Infrastructure/Persistence/Repository/Mapper/DbalBatchActionMapper.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\BatchAction\Infrastructure\Persistence\Repository\Mapper;
+
+use Ergonode\BatchAction\Domain\Entity\BatchAction;
+
+class DbalBatchActionMapper
```

Comment:
I think it would be clean to have symmetric operations 'mapper' or 'factory' - now it's scattered between two classes.